src/chains/ethereum/transaction/tests/index.test.ts

```diff
@@ -169,6 +170,65 @@ describe("@ganache/ethereum-transaction", async () => {
         const txFromRpc = TransactionFactory.fromRpc(tempAccessListTx, common);
         assert.strictEqual(txFromRpc.type.toString(), "0x0");
       });
+
+      describe("EIP-2", () => {
+        it("rejects transactions with too-high S values only when EIP-2 is active", () => {
+          const genesis = Common.forCustomChain(
+            "mainnet",
+            {
+              name: "ganache",
+              chainId: 1,
+              comment: "Local test network",
+              bootstrapNodes: []
+            },
+            // EIP-2 was in homestead, the first hardfork, so we need to create
+            // a common at the genesis (eke chainstart) so we can test at a fork
```

Comment:
```suggestion
            // a common at the genesis (aka chainstart) so we can test at a fork
```
Unless `eke` means something I'm not aware of?